### DISCUSSION TITLE
Github Orgs

### DISCUSSION THREAD

**Discussion by fangmarks** [BetaHuhn/deploy-to-vercel-action#22] [Q&A] [answered]
Would this Action be able to deploy repos I have in Github Organizations on my personal Vercel Account?

**Comment by BetaHuhn** [comment, marked as answer]
Hey @hokkqi!

I haven't tried it with a GitHub Org myself, but it should work just fine. The instructions in the [README](https://github.com/BetaHuhn/deploy-to-vercel-action#%EF%B8%8F-configuration) should apply to both personal and organization repositories, as there isn't really any difference when interacting with them.

On the Vercel side you just have specify the `VERCEL_ORG_ID` and `VERCEL_PROJECT_ID` of your [personal Vercel Project](https://github.com/BetaHuhn/deploy-to-vercel-action#vercel-project) (even personal projects have a ORG id) as well as a [`VERCEL_TOKEN`](https://github.com/BetaHuhn/deploy-to-vercel-action#tokens) which has access to your personal Vercel account.

Let me know if you encounter any problems or if there's anything else I can help you with!

**Reply by fangmarks** [reply]
I'll try it out sometime! Ty!

**Reply by fangmarks** [reply]
I currently have [this](https://github.com/thaldrin/primis/blob/master/.github/workflows/deploy.yml) as my deploy workflow, but I noticed that it doesn't actually respect the fact that there's a git submodule.

Do you think there's a way to have the action pull git submodules before it deploys to vercel?

**Reply by BetaHuhn** [reply]
The Action doesn't do anything to your files in the repo, it just deploys them as is.

You could run another Action before this one though, maybe [checkout-submodules](https://github.com/marketplace/actions/checkout-submodules) or since you already use [actions/checkout](https://github.com/actions/checkout) set the input `submodules` to true (see the [example](https://github.com/actions/checkout#usage) in their README).

I haven't tested this myself, so let me know if this works for you.

**Reply by fangmarks** [reply]
that worked fine! ty
